omas/omas_cocos.py

```diff
@@ -338,8 +338,8 @@
 _cocos_signals['divertors.divertor.:.power_currents.data']='?'                                           #[ADD?]# 1.750000 # current  [W]
 
 # EC_LAUNCHERS
-_cocos_signals['ec_launchers.beam.:.launching_position.phi']='?'                 #[ADD?]# 2.000000 # phi  [rad]
-_cocos_signals['ec_launchers.beam.:.steering_angle_pol']='?'                     #[ADD?]# 2.000000 # _pol  [rad]
+_cocos_signals['ec_launchers.beam.:.launching_position.phi']='TOR'                      # 2.000000 # phi  [rad]
+_cocos_signals['ec_launchers.beam.:.steering_angle_pol']=None                           # 2.000000 # _pol  [rad]
```

Comment:
I find this odd to have no COCOS dependence.

Reply:
hmm... yeah. Maybe it should be 'POL' now that you mention it.

Reply:
@jmcclena the definition is: "Steering angle of the EC beam in the R,Z plane (from the -R axis towards the -Z axis), angle_pol=atan2(-k_Z,-k_R), where k_Z and k_R are the Z and R components of the mean wave vector in the EC beam". The COCOS definition for the poloidal angle is starting from the +R axis (and COCOS 11 has the polodal angle to be clockwise). This means that the `steering_angle_pol` definition in IMAS is not consistent with COCOS 11. I would just leave it as `None`.

Reply:
So should we merge this then?